container/src/main/java/org/openremote/container/web/CORSConfig.java

```diff
@@ -118,7 +118,11 @@ public static Set<String> getCorsAllowedOriginsFromConfig() {
         String allowedOriginsStr = Config.getString(OR_WEBSERVER_ALLOWED_ORIGINS, null);
 
         if (!TextUtil.isNullOrEmpty(allowedOriginsStr)) {
-            allowedOrigins.addAll(Arrays.stream(allowedOriginsStr.split(",")).toList());
+            if (allowedOriginsStr.equals(DEFAULT_CORS_ALLOW_ALL)) {
```

Comment:
You're testing for equality, so what if allowedOriginStr is something like "*, xyz" ?
Would not match this and still result in an allow all + something else

Reply:
Discussed and this is considered an invalid configuration, we don't automatically fallback to only wildcard in that scenario.